atomic_reactor/util.py

```diff
@@ -785,3 +787,71 @@ def get_config_from_registry(image, registry, digest, insecure=False,
     logger.debug('Image %s:%s has config:\n%s', context, tag, blob_config)
 
     return blob_config
+
+def df_parser(df_path, workflow=None, cache_content=False, env_replace=True, parent_env=None):
```

Comment:
@lcarva I looked at writing a test where I pass in the Workflow and extract the `parent_env` as well as passing the `parent_env` by hand but I realized that the assertions on those tests will fail or pass depending on the version of `dockerfile-parse` at least until the version with support for `parent_env` is officially released.